themes/tfjs/source/js/codeSnippets.js

```diff
@@ -176,9 +272,30 @@ function initCodeBlocks(selector) {
       executeCodeSnippet(consoleLogElement, snippetText);
     });
 
-    consoleEditElement.addEventListener('click', function() {
+    consoleEditElement.addEventListener('click', function () {
       makeEditable(block);
       this.disabled = true;
     });
   });
+
+  function setBackend() {
+    const backendButtons = document.querySelectorAll('input[name="backend"]');
+    for (const backendButton of backendButtons) {
+      if (backendButton.checked) {
+        tf.setBackend(backendButton.value);
```

Comment:
Use `await tf.setBackend(backendButton.value);`? since webgpu is an async backend?

Reply:
Every run, it will call tf.ready, so  I think await before tf.setBackend is not necessary.
https://github.com/tensorflow/tfjs-website/pull/503/files#diff-d37fe47bb4c20101e1eef73668f1eaab8d9c62e2a90ea7cad50d72edfc3654d7R270